x-pack/plugins/cases/server/client/attachments/add.ts

```diff
@@ -231,53 +53,19 @@ async function getCombinedCase({
   logger: Logger;
   lensEmbeddableFactory: LensServerPluginSetup['lensEmbeddableFactory'];
 }): Promise<CommentableCase> {
-  const [casePromise, subCasePromise] = await Promise.allSettled([
-    caseService.getCase({
-      unsecuredSavedObjectsClient,
-      id,
-    }),
-    ...(ENABLE_CASE_CONNECTOR
-      ? [
-          caseService.getSubCase({
-            unsecuredSavedObjectsClient,
-            id,
-          }),
-        ]
-      : [Promise.reject(new Error('case connector feature is disabled'))]),
-  ]);
-
-  if (subCasePromise.status === 'fulfilled') {
-    if (subCasePromise.value.references.length > 0) {
-      const caseValue = await caseService.getCase({
-        unsecuredSavedObjectsClient,
-        id: subCasePromise.value.references[0].id,
-      });
-      return new CommentableCase({
-        logger,
-        collection: caseValue,
-        subCase: subCasePromise.value,
-        caseService,
-        attachmentService,
-        unsecuredSavedObjectsClient,
-        lensEmbeddableFactory,
-      });
-    } else {
-      throw Boom.badRequest('Sub case found without reference to collection');
-    }
-  }
+  const caseInfo = await caseService.getCase({
+    unsecuredSavedObjectsClient,
+    id,
+  });
 
-  if (casePromise.status === 'rejected') {
-    throw casePromise.reason;
-  } else {
-    return new CommentableCase({
-      logger,
-      collection: casePromise.value,
-      caseService,
-      attachmentService,
-      unsecuredSavedObjectsClient,
-      lensEmbeddableFactory,
-    });
-  }
+  return new CommentableCase({
```

Comment:
Do we need the `CommentableCase` any more?

Reply:
Probably not, should we move the functionality to the services? Should we do that in a follow up PR?

Reply:
I added it to the list in this issue: https://github.com/elastic/kibana/issues/123759